src/views/Stopwatch.vue

```diff
@@ -73,7 +76,10 @@ export default {
     },
     stoppedTimers() {
       return _.filter(this.events, (e) => (!e.data.running))
-    }
+    },
+    timersByDate() {
```

Comment:
Don't you also need to sort them as well?
When I edit a timer with a date that is older than the rest it always shows up in the top, then when I refresh the page it moves to the bottom as expected.
Should be fixed somehow.

Reply:
Ah yes, nice find. Need to v-for over the sorted keys.